xarray/__init__.py

```diff
@@ -52,6 +52,12 @@
     # Disable minimum version checks on downstream libraries.
     __version__ = "999"
 
+try:
+    from datatree import DataTree, open_datatree, register_datatree_accessor
+except ImportError:
+    ...
```

Comment:
There is almost certainly a better way to make `from xarray import DataTree` work than this

Reply:
I think this is fine as a temporary solution.